package.json

```diff
@@ -167,6 +167,7 @@
     "react-docgen-typescript-loader": "3.7.2",
     "react-dom": "17.0.2",
     "react-element-to-jsx-string": "14.3.4",
+    "react-number-format": "^4.9.4",
```

Comment:
@tokarevfedor1991 Can we make the package to be the exact version? Without the "^" symbol.

Reply:
ok)